packages/duckietown_demos/launch/indefinite_navigation.launch

```diff
@@ -2,7 +2,7 @@
 <launch>
   <arg name="veh" default="$(env VEHICLE_NAME)"/>
   <arg name="demo_name" value="indefinite_navigation"/>
-	<arg name="verbose" default="false"/>
+	<arg name="verbose" default="true"/>
```

Comment:
are these verbose flats still needed, or were they just used for debugging?

Reply:
This verbose tag also limits only in indefinite navigation so it should not affect lf behavior/performance. We currently leave it to be true for the ease of debugging and testing on different duckiebots. Once we are RTM stage I will disable that.